userspace/libsinsp/user.h

```diff
@@ -26,9 +26,6 @@ limitations under the License.
 
 class sinsp;
 class sinsp_evt;
-#if defined(HAVE_PWD_H) || defined(HAVE_GRP_H)
-namespace libsinsp { namespace procfs_utils { class ns_helper; }}
-#endif
```

Comment:
Can't we just declare this unconditionally? If it's not _used_ when not available, then it should compile just fine.